Examples/FriendlyFlix/app/FriendlyFlixSDK/Sources/FriendlyFlixClient.swift

```diff
@@ -0,0 +1,88 @@
+import FirebaseDataConnect
+import Foundation
+
+public extension DataConnect {
+  static var friendlyFlixConnector: FriendlyFlixConnector = {
+    let dc = DataConnect.dataConnect(connectorConfig: FriendlyFlixConnector.connectorConfig)
```

Comment:
There is a slightly changed signature in the latest gen code changes. It adds an optional `callerSDKType` param.